manifests/config.pp

```diff
@@ -42,5 +48,9 @@
       content => to_yaml($puppetwebhook::app_cfg),
       notify  => Service['puppet_webhook'],
       ;
+    "${env_path}/webhook":
```

Comment:
To be consistent, IMO the file should be named `${env_path}/puppetwebhook` or  `${env_path}/puppet_webhook`.